mckhannah1901/Co-op-Blog-Project
# Ath 1103 edit blog post

### JIRA 
[ATH-1103: Edit Blog Posts](https://wpengine.atlassian.net/secure/RapidBoard.jspa?rapidView=522&projectKey=ATH&modal=detail&selectedIssue=ATH-1103)

### What has been done
A logged in user can now edit a blog post and the updated at time populates correctly

### How to Test
1. Verify that a non logged in user can only view blog posts but can't edit them
2. Log in as a user
3. View the list of posts
4. Select the edit button on the post you'd like to edit
5. Fill in the form and publish
6. If successful, a flash message will tell you the post has been updated successfully
